src/algorand/algorandEngine.ts

```diff
@@ -470,6 +476,41 @@ export class AlgorandEngine extends CurrencyEngine<
 
         break
       }
+      case 'axfer': {
+        const edgeTokenId = Object.keys(this.allTokensMap).find(
+          tokenId => this.allTokensMap[tokenId].currencyCode === currencyCode
+        )
+        if (edgeTokenId == null) throw new Error('Unrecognized asset')
+        const assetIndex: string | undefined =
+          this.allTokensMap?.[edgeTokenId]?.networkLocation?.assetIndex
+        if (assetIndex == null) throw new Error('Unrecognized asset')
+
+        rawTx = algosdk.makeAssetTransferTxnWithSuggestedParamsFromObject({
+          assetIndex: parseInt(assetIndex),
+          to: publicAddress,
+          from: this.walletInfo.keys.publicKey,
+          amount: BigInt(amount),
```

Comment:
This library requires native browser BigInt support to function? Do we need checks for this?

Reply:
That's handled in the checkEnvironment function defined in the info file same as Polkadot